components/omega/src/infra/IOStream.cpp

```diff
@@ -1645,6 +1645,15 @@ Error IOStream::readFieldData(
    }
    // For back compatibility, try to read again with old field name
    if (Err.isFail()) {
+      // Check for "Layer" in string, backwards compatibility requires replacing
```

Comment:
Can we move this replacement up just after OldFieldName is defined? That keeps all the renaming in one place.

Also, we don't need to Log the change so I'd remove LOG_INFO call.